pobi.md

```diff
@@ -1,41 +1,41 @@
-2. In the meantime it is closely monitoring the L1 by being directly connected to a L1 node.
-3. As soon as the previous rollup was added to a mined L1 block, the aggregator takes that Merkle proof, feeds it to the TEE who replies with a signed rollup containing a random nonce generated inside the enclave.
-4. All the other aggregators will be doing roughly the same thing at the same time.
-5. At this point (which happens immediately after the successful publishing of the previous rollup in the L1), every aggregator will have a signed rollup with a random nonce which they will gossip between them. The party with the lowest nonce wins. All the aggregators know this, and a new round starts.
-6. The winning aggregator has to create an ethereum transaction that publishes this rollup to L1.
+1. A new round starts from the point of view of an aggregator when it decides that someone has gossiped a winning rollup. At that point, it creates a new empty rollup structure, points it to the previous one, and starts adding transactions to it (which are being received from users or by gossip).
+2. In the meantime it closely monitors the L1 by being directly connected to a L1 node.
+3. As soon as the previous rollup was added to a mined L1 block, the aggregator takes that Merkle proof, feeds it to the TEE, who replies with a signed rollup containing a random nonce generated inside the enclave.
+4. All the other aggregators do roughly the same thing at the same time.
+5. At this point (which happens immediately after successfully publishing the previous rollup in the L1), every aggregator has a signed rollup with a random nonce which they gossip between them. The party with the lowest nonce wins. All the aggregators know this, and a new round starts.
+6. The winning aggregator has to create an Ethereum transaction that publishes this rollup to L1.
 
-Note that by introducing the requirement for proof of inclusion in the L1, the cadence of publishing the rollups to the block times is synchronised. Also, note that the hash of the L1 block used to prove to the TEE that the previous rollup was published will be added to the current rollup such that the management contract, and the other aggregators know whether this rollup was actually generated correctly.
+Note that by introducing the requirement for proof of inclusion in the L1, the cadence of publishing the rollups to the block times is synchronised. Also, note that the hash of the L1 block used to prove to the TEE that the previous rollup was published is added to the current rollup such that the management contract and the other aggregators know whether this rollup was generated correctly.
 
-This sequence is depicted in the following diagram:
+The following diagram depicts this sequence:
 ![node-processing](./images/node-processing.png)
 
 ### Notation
-There are six elements which define a rollup :
+There are six elements that define a rollup :
```

Comment:
Is it 6 or 4 elements? And the numbers appear to go 1,2,5,6. Could just be the way GH displays the changes tho.